libs/external-router/driver/testing/src/testing.service.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { TestBed } from '@angular/core/testing';
+import { TestScheduler } from 'rxjs/testing';
+
+import { DaffExternalRouterTestingDriver } from './testing.service';
+import { DaffExternalRouterDriverTestingConfig } from './config';
+import { DaffExternalRouterDriverTestingModule } from './testing.module';
+
+describe('@daffodil/external-router/driver/testing | DaffExternalRouterTestingDriver', () => {
+	let service: DaffExternalRouterTestingDriver;
+	let scheduler: TestScheduler;
+
+	const setupTest = (
+		configuration: DaffExternalRouterDriverTestingConfig = {},
+	) => {
+		TestBed.configureTestingModule({
+			imports: [DaffExternalRouterDriverTestingModule.forRoot(configuration)],
+		});
+		service = TestBed.get(DaffExternalRouterTestingDriver);
```

Comment:
should be `TestBed.inject`

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a